framework/Optimizers/Optimizer.py

```diff
@@ -480,8 +480,8 @@ def initialize(self,externalSeeding=None,solutionExport=None):
 
     self.mdlEvalHist = self.assemblerDict['TargetEvaluation'][0][3]
     # check if the TargetEvaluation feature and target spaces are consistent
-    ins  = self.mdlEvalHist.getParaKeys("inputs")
-    outs = self.mdlEvalHist.getParaKeys("outputs")
+    ins  = self.mdlEvalHist.getVarss("inputs")
```

Comment:
getVars

Reply:
Yeah, unless you want it fixed here, I'll fix it in the Optimizer merge request coming up.  Sorry that slipped in.